### PR TITLE
Explicitly recommend order-independent completion style

### PR DESCRIPTION
**Pull request by rdiaz02**
Mentioned in https://github.com/emacs-citar/citar/issues/791

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -58,7 +58,9 @@ In addition, the following packages are strongly recommended for the best experi
 3. [[https://github.com/oantolin/embark][Embark]] (contextual actions)
 4. [[https://github.com/minad/marginalia][Marginalia]] (annotations, and also candidate classification for Embark)
 
-We also recommend Emacs 28, as this package relies on two of its features, that greatly enhance the UI.
+In particular, if you want to narrow your search using authors, titles, etc (i.e., not only citation keys), you need to use a completion style that is order independent. For example, [[https://github.com/oantolin/orderless][Orderless]] with ~completion-styles~ set to ~(orderless basic)~ (see example in  https://github.com/oantolin/orderless#overview).
+
```
Comment (bdarcus) [inline README.org:62]: Looks good in general, but this isn't a complete sentence ;-)

File: README.org [hunk 2]
```diff
@@ -58,7 +58,9 @@ In addition, the following packages are strongly recommended for the best experi
 3. [[https://github.com/oantolin/embark][Embark]] (contextual actions)
 4. [[https://github.com/minad/marginalia][Marginalia]] (annotations, and also candidate classification for Embark)
 
-We also recommend Emacs 28, as this package relies on two of its features, that greatly enhance the UI.
+In particular, if you want to narrow your search using authors, titles, etc (i.e., not only citation keys), you need to use a completion style that is order independent. For example, [[https://github.com/oantolin/orderless][Orderless]] with ~completion-styles~ set to ~(orderless basic)~ (see example in  https://github.com/oantolin/orderless#overview).
```
Comment (bdarcus) [inline README.org:61]:
```suggestion
In particular, if you want to narrow your search using authors, titles, etc (i.e., not only citation keys), you need to use a completion style that is order independent; for example, [[https://github.com/oantolin/orderless][Orderless]] with ~completion-styles~ set to ~(orderless basic)~ (see example in  https://github.com/oantolin/orderless#overview).
```

File: README.org [hunk 3]
```diff
@@ -58,7 +58,9 @@ In addition, the following packages are strongly recommended for the best experi
 3. [[https://github.com/oantolin/embark][Embark]] (contextual actions)
 4. [[https://github.com/minad/marginalia][Marginalia]] (annotations, and also candidate classification for Embark)
 
-We also recommend Emacs 28, as this package relies on two of its features, that greatly enhance the UI.
+In particular, if you want to narrow your search using authors, titles, etc (i.e., not only citation keys), you need to use a completion style that is order independent; for example, [[https://github.com/oantolin/orderless][Orderless]] with ~completion-styles~ set to ~(orderless basic)~ (see example in  https://github.com/oantolin/orderless#overview).
+
+We also recommend Emacs 28, as this package relies on two of its features, that greatly enhance the UI.
```
Comment (bdarcus) [inline README.org:63]:
```suggestion
We also recommend Emacs 28 or newer. 
```

**Comment by bdarcus** [comment]
I made the minor change. Thanks!

**Comment by rdiaz02** [comment]
Thank you; and sorry for it no being a complete sentence.

**Comment by bdarcus** [comment]
No hay problema!

**Pull request merged**